### PR TITLE
Release 2021.05 highlights

### PR DESCRIPTION
**Pull request by mcarrano**
This PR adds release highlights for milestone 2021.05.

@nicolethoen @evwilkin can you please review when you have a chance?

### CODE REVIEW

**Review by nicolethoen — APPROVED** [review]
LGTM

File: release-highlights/release-highlights-2021.05/release-highlights.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+![code block component](./img/code-block.png)
+
+Introduces a read-only code block component that is lighter weight than the full code editor for displaying a snippet of code with optional actions. The ability to show or hide lines is available for long code snippets.
+
+### [Optional item counts in a Select menu](https://www.patternfly.org/v4/components/select#checkbox-input-with-counts)
+
+![item counts in a select menu](./img/select-item-counts.png)
+
+Adds optional right-aligned item counts to a Select menu. This is useful for faceted filtering use cases where you many want to report the number of items matching each specific value.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.05/release-highlights.md:28]:
```suggestion
Adds optional right-aligned item counts to a Select menu. This is useful for faceted filtering use cases where you may want to report the number of items matching each specific value.
```

File: release-highlights/release-highlights-2021.05/release-highlights.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+See the [latest release notes](https://www.patternfly.org/v4/developer-resources/release-notes) for a more detailed list of changes.
+
+-----------------------------------------------------------------------------
+
+## What we’re working on...
+
+### 2021.06 (May 14)
+
+* [Log viewer component](https://github.com/patternfly/patternfly-react/issues/5341) - Introduce a native log viewer native beta component to display real-time streamed logs or static log data.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.05/release-highlights.md:54]:
```suggestion
* [Log viewer component](https://github.com/patternfly/patternfly-react/issues/5341) - Introduce a native log viewer beta component to display real-time streamed logs or static log data.
```

File: release-highlights/release-highlights-2021.05/release-highlights.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+* [New List variants](https://github.com/patternfly/patternfly-react/issues/5336) - Add new variants of the List component to include horizontal separators, icons and 'pf-m-plain' modifier.
+
+* [Composable dropdown menus](https://github.com/patternfly/patternfly-react/issues/5073) - Create a demo that shows how the Menu and Toggle Button components can be combined to create custom dropdown elements.
+
+* [Uncontrolled behavior for tabs](https://github.com/patternfly/patternfly-react/issues/4458) - To be more developer friendly, the Tabs component will support uncontrolled behavior such that the developer does not need to control which tab is the active one.
+
+### 2021.07 (June 4)
+
+* [Auto-complete search input](https://github.com/patternfly/patternfly-react/issues/5499) - Will introduce an option to add auto-complete behavior to the Search Input component.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.05/release-highlights.md:66]:
```suggestion
* [Auto-complete search input](https://github.com/patternfly/patternfly-react/issues/5499) - Will introduce an option to add auto-complete behavior to the Search input component.
```

File: release-highlights/release-highlights-2021.05/release-highlights.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+### 2021.06 (May 14)
+
+* [Log viewer component](https://github.com/patternfly/patternfly-react/issues/5341) - Introduce a native log viewer native beta component to display real-time streamed logs or static log data.
+
+* [Update toggle group styling](https://github.com/patternfly/patternfly/issues/3951) - Update selected color for a set of toggle buttons to use a lighter blue.
+
+* [New List variants](https://github.com/patternfly/patternfly-react/issues/5336) - Add new variants of the List component to include horizontal separators, icons and 'pf-m-plain' modifier.
+
+* [Composable dropdown menus](https://github.com/patternfly/patternfly-react/issues/5073) - Create a demo that shows how the Menu and Toggle Button components can be combined to create custom dropdown elements.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.05/release-highlights.md:60]: Is the capitalization correct here?

File: release-highlights/release-highlights-2021.05/release-highlights.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+* [Uncontrolled behavior for tabs](https://github.com/patternfly/patternfly-react/issues/4458) - To be more developer friendly, the Tabs component will support uncontrolled behavior such that the developer does not need to control which tab is the active one.
+
+### 2021.07 (June 4)
+
+* [Auto-complete search input](https://github.com/patternfly/patternfly-react/issues/5499) - Will introduce an option to add auto-complete behavior to the Search Input component.
+
+* [Add help text support to description list items](https://github.com/patternfly/patternfly-react/issues/5482) - Add the ability to open help in a popover by clicking of a field label in a description list.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.05/release-highlights.md:68]:
```suggestion
* [Add help text support to Description list items](https://github.com/patternfly/patternfly-react/issues/5482) - Add the ability to open help in a popover by clicking of a field label in a Description list.
```

File: release-highlights/release-highlights-2021.05/release-highlights.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+* [Add "visited" styling for links](https://github.com/patternfly/patternfly/issues/3726) - add visited styling to optionally apply to links on a page.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.05/release-highlights.md:70]:
```suggestion
* [Add "visited" styling for links](https://github.com/patternfly/patternfly/issues/3726) - Add visited styling to optionally apply to links on a page.
```

File: release-highlights/release-highlights-2021.05/release-highlights.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+* [Add "view more" action to a Select menu](https://github.com/patternfly/patternfly-react/issues/5647) - Add a View more action at the button of a select menu to enable incremental loading of long option lists.
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.05/release-highlights.md:72]:
```suggestion
* [Add "view more" action to a Select menu](https://github.com/patternfly/patternfly-react/issues/5647) - Add a view more action to the button of a select menu to enable incremental loading of long option lists.
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
Some nits mostly around capitlization - thanks!

File: release-highlights/release-highlights-2021.05/release-highlights.md (hunk 4 above)
Comment (mcarrano) [reply release-highlights/release-highlights-2021.05/release-highlights.md]: Actually should be "Menu toggle" and not "Toggle button." Fixed.

**Comment by mcarrano** [comment]
Good eye for catching all of these @evwilkin . Capitalization makes me crazy. Should all be fixed. Please look again when you have a chance.

**Review by evwilkin — APPROVED** [review]
LGTM

**Pull request merged**